### DISCUSSION TITLE
TurboWarp Extension Creator

### DISCUSSION THREAD

**Discussion by samuellouf** [TurboWarp/extensions#1118] [Show and tell]
Hi there!
I want to show all of you a little project I've been working on.
I created a TurboWarp Extension Creator.
It's a tool to create extensions more easily.
The only things you need to code are the blocks' functions.
You can use it online, offline, and even download it.
Click [here](https://samuellouf.github.io/TurboWarp-Extension-Creator/) to open the website, [here](https://samuellouf.github.io/TurboWarp-Extension-Creator/app/) to open the online app, and [here](https://samuellouf.github.io/TurboWarp-Extension-Creator/?download) to download the app.

**Comment by mcnole25** [comment]
I made something similar: [https://mcnole25.github.io/unsandbox-maker.html](url)

**Comment by scratchernotfounded** [comment]
Thank you very much man that's what I wanted but how do you add an inbox to the block

**Reply by scratchernotfounded** [reply]
I found it

**Comment by WAYLIVES** [comment]
Wow, this is really cool!!!

**Reply by samuellouf** [reply]
Thanks !

**Comment by Jax3dJaxHahaPlanetBall** [comment]
Uh I Will Still Use Turbobuilder, I Request Somthing Like Scratch, Where You Put Blocks In Like A Editor.

**Reply by samuellouf** [reply]
That's perfectly fine. I haven't updated TEC in a long time.

**Reply by Jax3dJaxHahaPlanetBall** [reply]
Oh Waittttttttt You Can Upload Extensions? YESSSSSS!!!

**Comment by samuellouf** [comment]
Surprise !
![NEW GUI](https://github.com/user-attachments/assets/84ba2899-2c62-40e7-937a-d4b3278d0526)
I'm updating TurboWarp Extension Creator.
I'll add :
- A new, fresher gui
- Ability to save/reimport projects
- Button block

**Comment by samuellouf** [comment]
I released the update!
See #2161.
